src/components/ShoppingCart.js

```diff
@@ -42,21 +42,23 @@ const ShoppingCart = ({
         }}
       >
         <section id="cart-header">
-          <h3>Your Cart</h3>
+          <h3 style={{ paddingTop: "10px" }}>Your Cart</h3>
           <button className="cart-x-button" onClick={() => setIsOpen(!isOpen)}>
-            X
+            x
           </button>
         </section>
-        <div>
+        <div style={{ padding: "10px" }}>
           {cart.map((p) => (
             <div key={p.id}>
```

Comment:
try making this div as a `flex` to have image on the left, and other details on the right. 

It's currently looks like this if you add more than 1 item: 
![Screenshot 2023-04-12 at 6 58 15 PM](https://user-images.githubusercontent.com/36715832/231627846-ff095348-3113-4301-acca-dce00ba80203.png)

Reply:
got that fixed.